app/templates/serviceLearning/slcProposal.html

```diff
@@ -91,6 +84,14 @@ <h2 class="text-center pt-4 pb-2"> New Proposal For Service-Learning Course</h2>
           <option value="{{ term.id }}" {{ "selected" if course.term == term }}> {{ term.description }}</option>
         {% endfor %}
       </select>
+      <label for="regularOccurenceToggle">How often will this course occur?</label>
```

Comment:
Need to change where the label points

Reply:
Also we might want to clarify that we are talking about recurrence, rather than how often the course meets or something. Not sure exactly how to clarify that, though